SPECS/python-hatch-fancy-pypi-readme/python-hatch-fancy-pypi-readme.spec

```diff
@@ -0,0 +1,103 @@
+Name:           python-hatch-fancy-pypi-readme
+Version:        24.1.0
+Release:        4%{?dist}
+Summary:        Hatch plugin for writing fancy PyPI readmes
+
+License:        MIT
+URL:            https://github.com/hynek/hatch-fancy-pypi-readme
+Source0:        https://files.pythonhosted.org/packages/b4/c2/c9094283a07dd96c5a8f7a5f1910259d40d2e29223b95dd875a6ca13b58f/${name}-${version}.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  python3-devel
+BuildRequires:  python3-hatchling
+BuildRequires:  python3-hatch-vcs
+BuildRequires:  python3-packaging
+BuildRequires:  python3-pathspec
+BuildRequires:  python3-pytest
+BuildRequires:  python3-pip
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-setuptools_scm
+BuildRequires:  python3-toml
+BuildRequires:  python3-trove-classifiers
+BuildRequires:  python3-wheel
+
+%global common_description %{expand:
+This provides a Hatch metadata plugin for everyone who cares about the
+first impression of their projectâ€™s PyPI landing page. It allows you to
```

Comment:
Can you fix the description?  the "projectâ€™s" has some character encoding issues.